kcauto-kai.sikuli/repair.sikuli/repair.py

```diff
@@ -254,11 +254,15 @@ def _update_combat_next_sortie_time(self):
         """Method to update the Combat module's next sortie timer based on the
         shortest timer in the internal repair timers list.
         """
-        self.repair_timers.sort()
-        shortest_timer = self.repair_timers[0]
+        if len(self.repair_timers) > 0:
+            if len(self.repair_timers) > 1:
+                self.repair_timers.sort()
+            shortest_timer = self.repair_timers[0]
+        else:
+            shortest_timer = datetime.now()
 
-        if shortest_timer > self.combat.next_combat_time:
+        if shortest_timer < self.combat.next_combat_time:
```

Comment:
This change should be reversed. Again, the repair module should defer to the combat module in regards to shortening the combat timer, and only delay further.

This should be indented to be under the `if len(self.repair_timers) > 0:` conditional, too.